pkgs/by-name/tt/ttop/package.nix

```diff
@@ -2,13 +2,13 @@
 
 buildNimPackage (finalAttrs: {
   pname = "ttop";
-  version = "1.2.8";
+  version = "1.5.2";
 
   src = fetchFromGitHub {
     owner = "inv2004";
     repo = "ttop";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-QMUrA3OjxlDa1OxptJL7T3SPDTzSwVz6zz+ueh9eovM=";
+    hash = "sha256-4jkVNfyEJ3CdCVe9z+bfj/63AU+HhNYgH4OEj1rl9Xo=";
```

Comment:
```
error: hash mismatch in fixed-output derivation '/nix/store/cr1c7ay8ybglr11fq2xf1kqk8i6vjilp-source.drv':
         specified: sha256-4jkVNfyEJ3CdCVe9z+bfj/63AU+HhNYgH4OEj1rl9Xo=
            got:    sha256-/rs5JjTXxptVHXL3fY8qP6Be3r5N871CEbUH7w6zx4A=
```

Reply:
It looks like upstream just re-tagged 1.5.2 :grimacing: https://github.com/inv2004/ttop/releases/tag/v1.5.2

It previously pointed at https://github.com/inv2004/ttop/commit/f0bc9c5663ca7aff679127d3340c77d0133ebe34 but was updated to https://github.com/inv2004/ttop/commit/214fcd123d03798c90b74f3b03b1556678fc7aab (comparison [here](https://github.com/inv2004/ttop/compare/f0bc9c5663ca7aff679127d3340c77d0133ebe34...214fcd123d03798c90b74f3b03b1556678fc7aab)).

I've updated the hash.